atspi-common/src/cache.rs

```diff
@@ -37,15 +37,15 @@ impl Default for CacheItem {
 	fn default() -> Self {
 		Self {
 			object: ObjectRef {
-				name: ":0.0".into(),
+				name: ":0.0".try_into().unwrap(),
```

Comment:
I think it's acceptable to use `from_static_str` in cases like this. There's no need to bring error handling when the value is known at compile time.